tools/dgeni/processors/filter-export-aliases.ts

```diff
@@ -22,27 +20,12 @@ export class FilterExportAliases implements Processor {
   $runBefore = ['categorizer'];
 
   $process(docs: DocCollection) {
-    return docs.filter(doc => this.filterAliasExport(doc));
+    return docs.filter(doc => {
+      // If the alias symbol and the actual resolved symbol have the same name, then this doc
+      // shouldn't be filtered. This check is necessary, because technically as per TypeScript,
+      // explicit and individual re-exports are being considered as aliases.
+      // For example: export {Test} from './my-file`;
+      return !(doc.aliasSymbol && doc.aliasSymbol.name !== doc.symbol.name);
```

Comment:
This could have been made _without_ the *parentheses*, but I wanted to make clear what is considered a aliased export.